packages/sns/package.json

```diff
@@ -38,5 +37,8 @@
   ],
   "peerDependencies": {
     "@dfinity/utils": "^0.0.6"
+  },
+  "dependencies": {
+    "js-sha256": "^0.9.0"
```

Comment:
why do we need this lib?

a `sha256` can be calculated without lib so not sure why we need it?

and if we really need it, is there any alternative? that lib has not been updated in 5 years

Reply:
Thanks for the explanation!

To be honest I would rather like not to introduce a dependency, I know the drill, that's the kind of stuffs that stay ultimately for ever.

Before we do so can you have a look at this code I found on SO and use in Papy, would that be applicable here?

https://github.com/papyrs/ic/blob/4298bc2f0e430548b1d4d80f4a3b4474c177d514/providers/ic/src/utils/crypto.utils.ts#L1

Reply:
done